### PR TITLE
Update README files for new directory layout

### PR DESCRIPTION
**Pull request by tjohns**

### CODE REVIEW

File: samples/android/gcm-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+This sample uses the Gradle build system. To build this project, use the
+"gradlew build" command or use "Import Project" in Android Studio. To use the
+app you will need a sender ID to be able to register and test client side APIs,
+and an API key to be able to test server side APIs. For more information, see
+[Implementing GCM Client on Android](https://developers.google.com/cloud-messaging/android/client).
+
+## Screenshots
+
+![Menu](samples/android/gcm-demo/screenshots/menu.png "The app's menu")
```
Comment (silvolu) [inline samples/android/gcm-demo/README.md]: The screenshot are still broken:
https://github.com/tjohns/gcm/tree/master/samples/android/gcm-demo

File: README.md [hunk 2]
```diff
@@ -1,45 +1,18 @@
...
+Google Cloud Messaging (GCM) is a service that lets developers send data from
+servers to users' devices, and receive messages from devices on the same
+connection. The service provides a simple, lightweight mechanism that servers
+can use to tell mobile applications to contact the server directly to fetch
+updated application user data. The GCM service handles all aspects of queueing
+of messages and delivery to client applications running on target devices.
 
-This project contains an Android app that demonstrates these aspects of GCM:
+This project contains client libraries and samples to help developers interface
+with and explore the Google Cloud Messaging APIs.
```
Comment (silvolu) [inline README.md:12]: Do you want to add a reference to the quickstarts here as well?

Comment (tjohns) [inline README.md:12]: Done.

File: samples/android/gcm-demo/README.md (hunk 1 above)
Comment (tjohns) [inline samples/android/gcm-demo/README.md]: Done.

**Pull request merged**